src/controller/jobRoleController.ts

```diff
@@ -0,0 +1,16 @@
+import type { Request, Response } from "express";
+import type { JobRole } from "../model/jobRole";
+import { getJobRoles } from "../service/jobRoleService";
+
+export namespace JobRoles {
+    export async function get(req: Request, res: Response): Promise<void> {
```

Comment:
Missing tests for this file

Reply:
Added `test/unit/controller/jobRoleControllerTest.ts`